docs/installation.md

```diff
@@ -5,56 +5,66 @@ description: How to install Yazi on various operating systems.
 
 # Installation
 
-Prerequisites:
-
-- file (required, for file type detection)
-- nerd-fonts ([_optional_](./faq.md#i-dont-like-nerdfonts))
-- ffmpegthumbnailer (_optional_, for video thumbnails)
-- unar (_optional_, for archive preview)
-- jq (_optional_, for JSON preview)
-- poppler (_optional_, for PDF preview)
-- fd (_optional_, for file searching)
-- rg (_optional_, for file content searching)
-- fzf (_optional_, for quick file subtree navigation)
-- zoxide (_optional_, for historical directories navigation)
+To use Yazi, you must have the following prerequisites installed:
+
+- [file](https://github.com/file/file) (for file type detection)
+
+Yazi can be _optionally_ extended with other command line tools to enable additional features.
+
+- [nerd-fonts](https://www.nerdfonts.com/) ([_recommended_](./faq.md#i-dont-like-nerdfonts))
+- [`ffmpegthumbnailer`](https://github.com/dirkvdb/ffmpegthumbnailer) (for video thumbnails)
+- [`unar`](https://theunarchiver.com/command-line) (for archive preview)
+- [`jq`](https://jqlang.github.io/jq/) (for JSON preview)
+- [`poppler`](https://poppler.freedesktop.org/) (for PDF preview)
+- [`fd`](https://github.com/sharkdp/fd) (for file searching)
+- [`rg`](https://github.com/BurntSushi/ripgrep) (for file content searching)
+- [`fzf`](https://github.com/junegunn/fzf) (for quick file subtree navigation)
+- [`zoxide`](https://github.com/ajeetdsouza/zoxide) (for historical directories navigation)
 
 ## Arch Linux
 
-```bash
+```sh
 sudo pacman -S yazi ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-If you want to use the latest git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
+If you want to use the latest Git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
 
-```bash
+```sh
 paru -S yazi-git ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-## macOS
+## macOS / Homebrew
+
+First, make sure that Homebrew is fully up-to-date with `brew update`.
+
+Then you can install Yazi by itself:
```

Comment:
```suggestion
Then you can install Yazi itself by:
```

Reply:
That has a different meaning. I was trying to show that you can install the optional dependencies separately.